tools/p11ne-db

```diff
@@ -61,12 +69,19 @@ cmd_pack-key() {
     enc_key=$(aws kms encrypt --key-id $key_id --plaintext "$key" --region $region --query CiphertextBlob --output text)
     ok_or_die "Failed to encrypt the key"
 
+    # Read the cert (optional)
+    local cert
+    if [ -n "$cert_file" -a -f "$cert_file" ]; then
+        cert=$(<"$cert_file")
+    fi
+
     local out_data
     out_data=$( jq -Rn \
             --arg a "$enc_key" \
             --arg b "$id" \
             --arg c "$label" \
-            '[{encrypted_pem_b64: $a, id: $b | tonumber, label: $c}]' )
+            --arg d "$cert"
+            '[{encrypted_pem_b64: $a, id: $b | tonumber, label: $c, cert_pem: $d}] | with_entries( select( .value != "" ) ) ')
```

Comment:
This needs a `\` after the cert argument.

Also instead of `with_entries` we could use  `'[{encrypted_pem_b64: $a, id: $b | tonumber, label: $c, cert_pem: $d}] | del(.[][] | select(. == ""))')`